octue/mixins/serialisable.py

```diff
@@ -18,6 +18,10 @@ def __init__(self, *args, **kwargs):
         # Ensure it passes construction arguments up the chain
         super().__init__(*args, **kwargs)
 
+    @classmethod
+    def deserialise(cls, serialised_object):
```

Comment:
Do we need to call super().deserialise here?

Reply:
I don't think there should be a `deserialise` method in any of the other mixins or any possible superclass so I don't think so!